app/api/get-cache-key.js

```diff
@@ -0,0 +1,7 @@
+const getCacheKey = (path, headers) => {
+  return headers.email ? `${path}-internal` : `${path}-${headers.crn}`
```

Comment:
If there is an email then the request is from an internal user so we cache the internal user view.  If it's not then it's from an external customer so we cache the view relevant to the external user.